app/screens/settings/advanced/index.tsx

```diff
@@ -28,38 +29,32 @@ const getStyleSheet = makeStyleSheetFromTheme((theme) => {
 });
 
 const EMPTY_FILES: ReadDirItem[] = [];
-const EMPTY_SERVERS: string[] = [];
 
 type AdvancedSettingsProps = {
     componentId: string;
 }
 const AdvancedSettings = ({componentId}: AdvancedSettingsProps) => {
     const theme = useTheme();
     const intl = useIntl();
+    const serverUrl = useServerUrl();
     const [dataSize, setDataSize] = useState<number|undefined>(0);
     const [files, setFiles] = useState<ReadDirItem[]>(EMPTY_FILES);
-    const [serverUrls, setServerUrls] = useState<string[]>(EMPTY_SERVERS);
     const styles = getStyleSheet(theme);
 
     const getAllCachedFiles = async () => {
-        const {totalSize, files: cachedFiles, serverUrls: allServerUrls} = await getAllFilesInCachesDirectory();
+        const {totalSize, files: cachedFiles} = await getAllFilesInCachesDirectory(serverUrl);
         setDataSize(totalSize);
         setFiles(cachedFiles || EMPTY_FILES);
-        setServerUrls(allServerUrls || EMPTY_SERVERS);
     };
 
     const onPressDeleteData = preventDoubleTap(async () => {
         try {
             if (files.length > 0) {
-                const deletePromises = [];
-                for (const server of serverUrls) {
-                    deletePromises.push(deleteFileCache(server));
-                }
-                await Promise.all(deletePromises);
+                await deleteFileCache(serverUrl);
+                await getAllCachedFiles();
```

Comment:
`getAllCachedFiles` here is just to update the state, right?

Reply:
Right